core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-header/dot-experiments-ui-header.component.html

```diff
@@ -6,13 +6,20 @@
         <span data-testId="title">{{ title }}</span>
     </ng-container>
 
-    <ng-container *ngIf="!!status">
+    <ng-container *ngIf="!!experiment?.status">
         <div class="flex flex-grow-1 align-items-center justify-content-between">
             <p-tag
+                [value]="
+                    (experiment?.status | titlecase | dm) +
+                    (experiment?.status === experimentStatus.RUNNING
+                        ? ' ' +
+                          ('dot.common.until' | dm) +
+                          ' ' +
+                          (experiment?.scheduling.endDate | date : 'EEE, LLL dd, Y')
```

Comment:
I wonder if we can create a const for this `date`?

Reply:
agree, changed.